browser/brave_shields/ad_block_service_browsertest.cc

```diff
@@ -2158,7 +2153,6 @@ IN_PROC_BROWSER_TEST_F(AdBlockServiceTest, CosmeticFilteringIframeScriptlet) {
   ASSERT_EQ(true, EvalJs(contents, "show_ad"));
 }
 
-<<<<<<< HEAD
 // Test cosmetic filtering on an element that already has an `!important`
 // marker on its `display` style.
 IN_PROC_BROWSER_TEST_F(AdBlockServiceTest,
```

Comment:
`AdBlockServiceTest.CosmeticFilteringOverridesImportant` will have to be disabled as well, that was the primary issue fixed by the original PR

Reply:
@antonok-edm also CosmeticFilteringDynamicCustom doesn't work. Is it expected?

Reply:
Yeah, unfortunately it looks like I inserted two different fixes into the same PR. If you un-revert the changes in `CosmeticFilteringDynamicCustom` and in `test/data/cosmetic_filtering.html`, I think that one should work fine.

Reply:
In fact it's not enough..
```
[55080:56304:0411/175238.315:INFO:CONSOLE(5)] "still waiting for css selector", source: __const_std::string&_script__ (5)
content/public/test/browser_test_utils.cc(2659): error: Failed
RunLoop::Run() timed out. Timeout set at ProxyRunTestOnMainThreadLoop@content/public/test/browser_test_base.cc:809.
```

Reply:
I just pushed a couple of fixes for that as well.